src/System.Security.Cryptography.Primitives/tests/KeyedHashAlgorithmTests.cs

```diff
@@ -49,6 +49,8 @@ public void EnsureGetKeyCopies()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework,
+            "Change needs to be ported to netfx")]
```

Comment:
Assuming that netfx isn't nulling out the key in Dispose, I don't know we'd change that... seems like more compat risk than we take there.